tenable/io/v3/vm/remediation_scans/api.py

```diff
@@ -0,0 +1,338 @@
+'''
+Remediation Scans
+=================
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`Remediation scan create <io-scans-remediation-creates>` API endpoints.  # noqa: E501
+
+Methods available on ``tio.v3.vm.remediation_scans``:
+
+.. rst-class:: hide-signature
+.. autoclass:: RemediationScansAPI
+    :members:
+'''
+
+
+from typing import Dict, Union
+
+from requests import Response
+
+from tenable.errors import UnexpectedValueError
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.base.iterators.explore_iterator import (CSVChunkIterator,
+                                                           SearchIterator)
+from tenable.io.v3.vm.remediation_scans.schema import \
+    RemScansDocumentCreateSchema
+from tenable.utils import dict_merge
+
+
+class RemediationScansAPI(ExploreBaseEndpoint):
+    _path = 'api/v3/scans'
+    _conv_json = True
+
+    def search(self,
```

Comment:
Methods are not alphabetized.

Reply:
not strictly needed. we can ignore this arrangement.